.github/scripts/tests/tests.sh

```diff
@@ -30,13 +26,19 @@ DSPA_PATH="${GIT_WORKSPACE}/tests/resources/dspa-lite.yaml"
 DSPA_EXTERNAL_PATH="${GIT_WORKSPACE}/tests/resources/dspa-external-lite.yaml"
 CONFIG_DIR="${GIT_WORKSPACE}/config"
 RESOURCES_DIR_CRD="${GIT_WORKSPACE}/.github/resources"
-DSPO_IMAGE="${REGISTRY_ADDRESS}/data-science-pipelines-operator"
 OPENDATAHUB_NAMESPACE="opendatahub"
 RESOURCES_DIR_PYPI="${GIT_WORKSPACE}/.github/resources/pypiserver/base"
 # ------------------------------------
 
 # ------------------------------------
 # Functions
+get_dspo_image() {
+  if [ "$REGISTRY_ADDRESS" = "" ]; then
+      echo "REGISTRY_ADDRESS variable not defined." && exit 1
+  fi
+  echo "${REGISTRY_ADDRESS}/data-science-pipelines-operator"
```

Comment:
nit: can you provide more description in the echo statement

Reply:
The echo is how you return a value in a bash script

Reply:
I think you misunderstood. I'm saying to echo more than just the image, for example: 
```bash
echo "Using ${REGISTRY_ADDRESS}/data-science-pipelines-operator for DSPO image"  
```

Reply:
Done